src/rnnlm/rnnlm-utils-test.cc

```diff
@@ -162,12 +168,46 @@ void UnitTestSamplingTime(int iters) {
 }  // end namespace rnnlm
 }  // end namespace kaldi.
 
-int main() {
+int main(int argc, char **argv) {
   using namespace kaldi;
   using namespace rnnlm;
   int N = 10000;
   UnitTestSamplingConvergence();
   UnitTestSampleWithProbOne(N);
   UnitTestSamplingTime(N);
-}
 
+  const char *usage = "";
+  ParseOptions po(usage);
+  po.Read(argc, argv);
+  std::string arpa_file = po.GetArg(1), history_file = po.GetArg(2);
```

Comment:
this is probably OK now but you need to change it so it doesn't need cmd arguments